lib/account/account-manager.js

```diff
@@ -1,102 +1,171 @@
 "use strict";
+
 /**
  * Manages the business logic of account controls.
  */
 
 /**
- * The object sent in which has information needed to validate the user
- * has properly set up their two factor authentication and has generated
- * a password for use to save.
+ * Contains information needed to validate the user has
+ * properly set up their two factor authentication and
+ * has generated a password.
+ *
+ * @typedef {Object} accountManager~accountinfo
+ * @property {string} regId Unhashed registration id sent to client
+ * @property {string} currentMfa
+ * @property {string} previousMfa
+ * @property {string} password
```

Comment:
Shouldn't this also contain email?  We're not sending an email, but we should still accept one at this stage.